meka/srcs/file.c

```diff
@@ -232,7 +232,7 @@ bool    Load_ROM(t_load_mode load_mode, bool user_verbose)
         case LOAD_MODE_COMMANDLINE:
             Quit_Msg("%s\n\"%s\"\n", meka_strerror(), g_env.Paths.MediaImageFile);
             // Quit_Msg(meka_strerror());
-            return false;
+//            return false;

```

Comment:
Mistake, I turned on /w4 but didn't intend to PR it. (Unreachable code due to exit(1) in Quit_Msg().)